README.md

```diff
@@ -62,6 +62,7 @@ The main features of Gatus are:
 - **Metrics**
 - **Low resource consumption**: As with most Go applications, the resource footprint that this application requires is negligibly small.
 - **GitHub uptime badges**: ![Uptime 1h](https://status.twinnation.org/api/v1/badges/uptime/1h/core_twinnation-external.svg) ![Uptime 24h](https://status.twinnation.org/api/v1/badges/uptime/24h/core_twinnation-external.svg) ![Uptime 7d](https://status.twinnation.org/api/v1/badges/uptime/7d/core_twinnation-external.svg)
+- **API**: Service health can be pulled into any custom application by leveraging the built in JSON API.

```

Comment:
```suggestion
```
While the API is a nice feature, I don't think it's attractive enough to be explicitly specified in this section, which is mostly there to convince people to use Gatus.
This is because this